app/src/main/kotlin/com/twain/interprep/domain/usecase/resourceLink/UpdateResourceLinkUseCase.kt

```diff
@@ -0,0 +1,10 @@
+package com.twain.interprep.domain.usecase.resourceLink
+
+import com.twain.interprep.data.model.ResourceLink
+import com.twain.interprep.domain.repository.ResourceLinkRepository
+
```

Comment:
This method is not used. Also, UpdateNoteUseCase is never used too. Since we "update" notes by inserting the each of notes in the database first and inserting all the notes into the database again when the user clicks the back button. In other words, we rely on the replacing policy when inserting the note to "update" the note in the database.

Reply:
We will create another pr to replace update and insert Dao method to upsert